### PR TITLE
PathwayMatcher 1.9.1 updates

### PR DESCRIPTION
**Pull request by CarlosHorro**
Some options changed.
SUBSET math criteria by default instead of SUPERSET when proteoforms match type is chosen. Updated test graphs accordingly.

### CODE REVIEW

File: tools/pathwaymatcher/test-data/proteoforms_graphs/proteoformInternalEdges.tsv [hunk 1]
```diff
@@ -1,79 +1,3 @@
 id1	id2	type	container_id	role1	role2
-Q15759;	Q15759;00047:180,00048:182	Reaction	R-HSA-1247960	input	output
-P00519;	Q15759;	Reaction	R-HSA-448957	input	input
-P00519;	Q15759;	Reaction	R-HSA-448957	input	output
-P00519;	Q15759;	Reaction	R-HSA-448957	output	input
-P00519;	Q15759;	Reaction	R-HSA-448957	output	output
-Q15759;	Q15759;00047:180,00048:182	Reaction	R-HSA-450333	input	output
-Q15759;	Q15759;00047:180,00048:182	Reaction	R-HSA-3238999	input	output
```
Comment (bgruening) [inline tools/pathwaymatcher/test-data/proteoforms_graphs/proteoformInternalEdges.tsv]: @CarlosHorro do these dramatic changes in the test data makes sense for you?

**Comment by bgruening** [comment]
Looks good to me. The results are just dramatically different, is that expected?

**Comment by CarlosHorro** [comment]
Hi @bgruening ,

I've asked the developer about this topic and he told me they make sense due to internal changes and an updated version of Reactome db into this version.

**Comment by bgruening** [comment]
@CarlosHorro perfect thanks. One last thing is geneExternalEdges.tsv actually used? It is huge and it would be nice to strip it down or remove it.

**Comment by CarlosHorro** [comment]
I didn't realise about its size, so as it's not mandatory I've removed it.

**Comment by CarlosHorro** [comment]
This should be ready to be merged I think :-)

**Comment by bgruening** [comment]
Merci!

**Pull request merged**